experimental/AlgebraicStatistics/src/GaussianGraphicalModels.jl

```diff
@@ -0,0 +1,412 @@
+###################################################################################
+#
+#       Gaussian Rings
+#
+###################################################################################
+
+
+
+struct GaussianRing
+    ring::Ring
+    gens::Dict
```

Comment:
Based on what you wrote below, maybe the type here can be strengthened?
```suggestion
    gens::Dict{Tuple{Int64, Int64}, <:PolyRingElem}
```

Reply:
I just committed a number of strengthenings to the types of `MarkovRing` and `GaussianRing`.